apps/mosaico/src/Article/Article.purs

```diff
@@ -385,10 +385,23 @@ render { props, state, setState } =
             { className: block <> " " <> block <> "__question"
             , children: [ DOM.text question ]
             }
+        Related related -> DOM.figure
+            { className: block <> " " <> block <> "__related"
+            , children:
+                [ DOM.ul_ $ map renderRelatedArticle related
+                ]
+            }
       where
         block = "article-element"
         isDiv = isElem "<div"
         isBlockquote = isElem "<blockquote"
         -- Does the string start with wanted element
         isElem elemName elemString =
           Just 0 == String.indexOf (Pattern elemName) elemString
+        renderRelatedArticle { uuid, title } =
+          DOM.li_
+            [ DOM.a
+                { href: "/artikel/" <> uuid
```

Comment:
Let's prevent default behavior of making a server side call and just write this to the router

Reply:
What about right click to open in new tab (or middle-click)? We can't get that without using `a href`, the router never sees that action. I think we should support that use.

Reply:
We don't need to remove the `a` tag, we just need to prevent the default behavior when we have js available. It does not seem to affect to middle clicks.
https://pursuit.purescript.org/packages/purescript-react-basic-dom/4.2.0/docs/React.Basic.DOM.Events#v:capture
```
DOM.a
  { href: "/artikel/" <> uuid
  , children: [ DOM.text title ]
  , onClick: capture_ $ Console.log "click"
  }
```

Reply:
Ok, right. I'll add `onArticleClick`, similar to what `onTagClick` is.